src/app/utils/codemirror/eslint-lint.js

```diff
@@ -367,6 +375,7 @@ export default (async function initialize() {
   while (!window.eslint) {
     // eslint-disable-next-line
     await delay(100);
+    window.require = origRequire;
```

Comment:
damn as I see now the placement is wrong here. I have to move it out of the loop. *shame*